models/User.go

```diff
@@ -1009,6 +1037,10 @@ func PlaceBidOrder(userId uint32, bid *Bid) (uint32, error) {
 		return errorHelper("Error while subtracting order fee from user. Rolling back. Error: %+v", err)
 	}
 
+	if err := AddUserReservedCash(user, reservedCash, tx); err != nil {
+		return errorHelper("Error while subtracting reserved cash from user. Rolling back. Error: %+v", err)
```

Comment:
How is this error message accurate? Please check what you're copy pasting. Update the error message.